jbmc/src/java_bytecode/ci_lazy_methods.cpp

```diff
@@ -292,13 +292,13 @@ bool ci_lazy_methodst::handle_virtual_methods_with_no_callees(
     }
 
     // Check that `get_virtual_method_target` returns a method now
-    const irep_idt &call_basename = virtual_function.get_component_name();
-    const irep_idt method_name = get_virtual_method_target(
-      instantiated_classes, call_basename, call_class, symbol_table);
-    CHECK_RETURN(!method_name.empty());
+    const irep_idt &method_name = virtual_function.mangled_method_name();
```

Comment:
:pick: keep one rename per commit in future

Reply:
:question: `method_name` and `method_id` is confusing as to what is the difference between them. Perhaps `method_name` and `resolved_method_symbol_id`?

Reply:
Would it be an improvement if I renamed the `method_name` variable to `mangled_method_name`?